PWGHF/D2H/DataModel/ReducedDataModel.h

```diff
@@ -435,33 +435,108 @@ DECLARE_SOA_COLUMN(MlScoreBkgProng0, mlScoreBkgProng0, float);             //! B
 DECLARE_SOA_COLUMN(MlScorePromptProng0, mlScorePromptProng0, float);       //! Prompt ML score of the D daughter
 DECLARE_SOA_COLUMN(MlScoreNonpromptProng0, mlScoreNonpromptProng0, float); //! Nonprompt ML score of the D daughter
 } // namespace hf_reso_cand_reduced
+namespace hf_reso_3_prong
+{
+DECLARE_SOA_COLUMN(PxProng0, pxProng0, float);                              
+DECLARE_SOA_COLUMN(PyProng0, pyProng0, float);
+DECLARE_SOA_COLUMN(PzProng0, pzProng0, float);
+DECLARE_SOA_COLUMN(PxProng1, pxProng1, float);                              
```

Comment:
Most of your columns already exist in `CandidateReconstructionTables`. Why do you need to duplicate them?

Reply:
No real need, I just wanted to have all the columns in the same place, but I can use the existing ones if needed